pyigm/metallicity/MCMC_output.py

```diff
@@ -0,0 +1,849 @@
+import numpy as np
```

Comment:
Please start this module off with a doc description
of what it is for.  Am not sure myself.